instance/static/html/instance/index.html

```diff
@@ -50,7 +50,13 @@
                 |
                 <span>Studio: <a href="{{ selected.instance.studio_url }}" target="_new">{{ selected.instance.studio_url }}</a></span>
               </li>
-              <li><a href="{{ selected.instance.github_branch_url }}" target="_new">{{ selected.instance.github_base_url }}</a> | {{ selected.instance.branch_name }}</li>
+              <li>
+                <a href="{{ selected.instance.github_branch_url }}" target="_new">{{ selected.instance.github_base_url }}</a> {{ selected.instance.branch_name }}
+                {% if selected.instance.github_pr_url %}
+                | <a href="{{ selected.instance.github_pr_url }}" target="_new">PR#{{ selected.instance.github_pr_number }}</a>
+                {% endif %}
+              </li>
+              {% endif %}
```

Comment:
@smarnach Is that showing up correctly on your development environment? This template isn't interpreted by the Django templating language, it uses the angular syntax, so `{% if ... %}` won't work - it shows this for me:

![screenshot-localhost 5000-2015-09-12-08-53-20](https://cloud.githubusercontent.com/assets/514483/9830384/5ca38748-592c-11e5-8f1e-3507a6e95fe3.png)

Instead you need to use the `ng-if` directive, like in https://github.com/open-craft/opencraft/blob/smarnach/add-pr-link/instance/static/html/instance/index.html#L33